lib/src/like_button.dart

```diff
@@ -374,6 +362,16 @@ class _LikeButtonState extends State<LikeButton> with TickerProviderStateMixin {
       clipper: LikeCountClip(),
     );
 
+    if (widget.countDecoration != null) {
```

Comment:
Maybe some comments can added to here such as:
`/// No decoration and padding will be created when [countDecoration] or [likeCountPadding] is null.`

Reply:
Really? :)

Reply:
> Really? :)

Of course, we should make this prettier.

Reply:
may be user doesn't want like widget but any other widget base on like count.